### ISSUE TITLE
Possible wrong implementation in loralib layers

### ISSUE THREAD

**Issue by jichan3751** [microsoft/LoRA#34]
Hi,

In loralib's layer modules, 
https://github.com/microsoft/LoRA/blob/33b953630763c6299d2349abc8f154a3951a7984/loralib/layers.py#L138

It seems like `eval() `function which merges W+BA is never called.

This is because when changing the model to evaluation mode in torch by calling `model.eval()`, submodules of the model calls `module.train(mode=False)`, not `module.eval()` 
https://pytorch.org/docs/stable/_modules/torch/nn/modules/module.html#Module

I think this may be a bug and the weight was never merging in the evaluation mode.
Can we make edits to implement current `eval()` function in `train(mode=False)`?

**Comment by edwardjhu** [comment]
Good catch!

I just pushed a fix to a new branch called `eval`.

Please let me know if this resolves the issue for you!
